packages/alias/alias.spec.mjs

```diff
@@ -1,4 +1,4 @@
-import assert from 'node:assert/strict';
+import { equal } from 'node:assert/strict';
```

Comment:
I think it's better to keep consistent with other tests, and `equal` is too generic—better to explicitly know it's an assertion.